plugins/woocommerce/src/Internal/Admin/ProductReviews/Reviews.php

```diff
@@ -0,0 +1,656 @@
+		global $comment;
+
+		// Bail out if not a text we should replace.
+		if ( ! in_array( $text, [ 'Edit Comment', 'Moderate Comment' ], true ) ) {
+			return $translation;
+		}
+
+		// Try to get comment from query params when not in context already.
+		if ( ! $comment && isset( $_GET['action'], $_GET['c'] ) && 'editcomment' === $_GET['action'] ) {
+			$comment_id = absint( $_GET['c'] );
+			$comment    = get_comment( $comment_id ); // phpcs:ignore WordPress.WP.GlobalVariablesOverride.Prohibited
+		}
+
+		$is_reply = false;
+
+		if ( isset( $comment->comment_parent ) && $comment->comment_parent > 0 ) {
+			$is_reply = true;
+			$comment = get_comment( $comment->comment_parent ); // phpcs:ignore WordPress.WP.GlobalVariablesOverride.Prohibited
+		}
+
+		// Only replace the translated text if we are editing a comment left on a product (ie. a review).
+		if ( isset( $comment->comment_post_ID ) && 'product' === get_post_type( $comment->comment_post_ID ) ) {
+			if ( 'Edit Comment' === $text ) {
+				$translation = $is_reply
+					? __( 'Edit Review Reply', 'woocommerce' )
+					: __( 'Edit Review', 'woocommerce' );
+			} elseif ( 'Moderate Comment' === $text ) {
+				$translation = $is_reply
+					? __( 'Moderate Review Reply', 'woocommerce' )
+					: __( 'Moderate Review', 'woocommerce' );
+			}
+		}
+
+		return $translation;
+	}
+
+	/**
+	 * Returns a new instance of `ReviewsListTable`, with the screen argument specified.
+	 *
+	 * @return ReviewsListTable
+	 */
+	protected function make_reviews_list_table() : ReviewsListTable {
+		return new ReviewsListTable( [ 'screen' => $this->reviews_page_hook ? $this->reviews_page_hook : 'product_page_product-reviews' ] );
+	}
+
+	/**
+	 * Initializes the list table.
+	 *
+	 * @return void
+	 */
+	protected function load_reviews_screen() : void {
+		$this->reviews_list_table = $this->make_reviews_list_table();
+		$this->reviews_list_table->process_bulk_action();
+	}
+
+	/**
+	 * Renders the Reviews page.
+	 *
+	 * @return void
+	 */
+	public function render_reviews_list_table() : void {
+
+		$this->reviews_list_table->prepare_items();
+
+		ob_start();
+
+		?>
+		<div class="wrap">
+			<h2><?php echo esc_html( get_admin_page_title() ); ?></h2>
+
+			<?php $this->reviews_list_table->views(); ?>
+
+			<form id="reviews-filter" method="get">
+				<?php $page = isset( $_REQUEST['page'] ) ? sanitize_text_field( wp_unslash( $_REQUEST['page'] ) ) : static::MENU_SLUG; ?>
+
+				<input type="hidden" name="page" value="<?php echo esc_attr( $page ); ?>" />
+				<input type="hidden" name="post_type" value="product" />
+
+				<?php $this->reviews_list_table->search_box( __( 'Search Reviews', 'woocommerce' ), 'reviews' ); ?>
+
+				<?php $this->reviews_list_table->display(); ?>
+			</form>
+		</div>
+		<?php
+		wp_comment_reply( '-1', true, 'detail' );
+		wp_comment_trashnotice();
+
+		/**
+		 * Filters the contents of the product reviews list table output.
+		 *
+		 * @since 6.6.0
```

Comment:
Here and elsewhere, we should update to `6.7.0` (we've entered code freeze for 6.6.0).